TritonSE/TAP-Foster-Dog-Matching
# Feature/parth4apple/manage dog profiles

### Administrative Info
[GitHub Issue #]
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
Almost finished the manage dog profiles page. 
TODO:
- Add image hosting and make corresponding changes to frontend

### Testing
How did you confirm your changes worked? 
Run the frontend and backend. Go to manage dog profiles page and experiment with the functions.
- Click on a create a new dog
- Click on view all/ view less
- Click on any of the dogs listed (and then click on edit)

### Confirmation of Change 

| [TODO Screenshot Title] |
| --- |
| [TODO Screenshot Link] |
